ecosystem/typescript/sdk/src/transaction_builder/aptos_types/rotation_proof_challenge.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+import { Serializer } from "../bcs";
+import { AccountAddress } from "./account_address";
+
+export class RotationProofChallenge {
```

Comment:
it would be nice to generate this class definition from the abi

Reply:
yeah. At the moment, both python and TS sdks are not using the sdk builder generator. The code generated was really bad. For TS, the code was buggy and incomplete. For python, the generated classes use `numpy`. In the future, would explore if we want to bring the sdk builder generator up to date for these two languages.